src/components/MenuItem.tsx

```diff
@@ -261,6 +261,12 @@ type MenuItemBaseProps = {
     /** Whether should render helper text as HTML or as Text */
     shouldParseHelperText?: boolean;
 
+    /** Whether should render hint text as HTML or as Text */
+    shouldParseHintText?: boolean;
```

Comment:
Please rename these to `shouldRenderHintAsHTML` and `shouldRenderErrorAsHTML`

Reply:
There are other props in this component that are named similary, are we sure we want to name it differently?
```
shouldParseTitle = false,
shouldParseHelperText = false,
shouldParseHintText = false,
shouldParseErrorText = false,
```

Reply:
but to be clear, I'm not asking you to rename the other props in this PR :D Maybe you or I could do a followup PR to clean those up later.